Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
```

Comment:
The body of for loop could be moved into a function. That would leave you with two options then:
A- just call the same function twice
B- do what you do, but in a body call a function instead.